dialme/README.md

```diff
@@ -30,7 +30,7 @@ This is a rather simple protocol that communicates a multiaddress to perform the
 message DialMe {
   message Peer {
     required bytes id = 1;    // peer id
-    repeated bytes addrs = 2; // array of multiadrs
```

Comment:
This commit should have changed `repeated` to `optional` (unless I'm missing something).